### PR TITLE
0.1 XYMs max fee default

### PR DESCRIPTION
**Pull request by fboucquez**
Replaced default Max Fee from the node's 'Min Fee Multiplier' to 0.1 XYMs.

Fixes https://github.com/nemtech/symbol-bootstrap/issues/241

### CODE REVIEW

File: presets/shared.yml [hunk 1]
```diff
@@ -85,6 +85,7 @@ votingKeysDirectory: ./votingkeys
 enableDelegatedHarvestersAutoDetection: true
 catapultAppFolder: /usr/catapult
 rewardProgramAgentPort: 7881
+announceDefaultMaxFee: 100000
```
Comment (Wayonb) [inline presets/shared.yml:88]: How did you reach that 0.1 as the default max fee?

File: src/service/AnnounceService.ts [hunk 2]
```diff
@@ -265,32 +257,26 @@ export class AnnounceService {
...
                     );
-                    let lockFundsTransaction: Transaction = LockFundsTransaction.create(
+                    const lockFundsTransaction: Transaction = LockFundsTransaction.create(
                         deadline,
                         currency.createRelative(10),
                         UInt64.fromUint(1000),
                         signedAggregateTransaction,
                         networkType,
-                        defaultMaxFee,
+                        maxFee,
```
Comment (Wayonb) [inline src/service/AnnounceService.ts:278]: The max fee for a lock fund tx will be a lot smaller than aggregate tx.  Is there an issue with calculating the max fee from the minFee multiplier?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: presets/shared.yml (hunk 1 above)
Comment (fboucquez) [reply presets/shared.yml:88]: It seems to be a sensible default. Cheap and fast enough.

File: src/service/AnnounceService.ts (hunk 2 above)
Comment (fboucquez) [reply src/service/AnnounceService.ts:278]: The min fee multiplier was more problematic than expected. It creates a max fee too low that doesn't confirm. Maybe this is a bug in the server? Pretty much everyone is using the --maxFee param. 

I've seen this problem a lot https://forum.nem.io/t/symbol-supernode-programme-community-testing/30081/16

**Pull request closed without merging**